huaweicloud/services/iam/resource_huaweicloud_identity_user_role_assignment.go

```diff
@@ -125,7 +125,7 @@ func resourceIdentityUserRoleAssignmentDelete(_ context.Context, d *schema.Resou
 	roleID := d.Get("role_id").(string)
 	enterpriseProjectID := d.Get("enterprise_project_id").(string)
 
-	err = eps_permissions.UserPermissionsCreate(client, enterpriseProjectID, userID, roleID).ExtractErr()
+	err = eps_permissions.UserPermissionsDelete(client, enterpriseProjectID, userID, roleID).ExtractErr()
```

Comment:
Cause the test case can also pass due to calling the wrong deletion interface. It is best to modify the test cases as well.